src/easypay/__init__.py

```
@@ -29,6 +29,7 @@
 from . import errors
 from . import mb
 
-from .base import BASE_URL, Scheduler, API, ShelveAPI
+from .base import BASE_URL, Scheduler, API, ShelveAPI, APIV2
```

Ruff (0.8.0)

`.base.BASE_URL` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)